Rakefile

```diff
@@ -10,4 +11,16 @@ Rake::TestTask.new do |t|
   t.test_files = FileList['test/**/*_test.rb']
 end
 
+task :test_interactive do
+  scripts = FileList['examples/*.rb', 'examples/test_method/*.rb']
+            .select { |name| FileTest.executable?(name) }
+  scripts.each do |script|
+    system('clear') || system('cls')
+    system("ruby -Ilib #{script}")
+    puts "\nExecuted '#{script}'"
+    puts 'Press a key to continue or ESC to exit.'
+    break if STDIN.getch == "\e"
+  end
+end
```

Comment:
This is nice temporary solution until we figure out how to actually automate tests. Huge pain, the library suffer from regressions.

Reply:
Any chance of putting Japanese tests aside (like a separate task or variable) because I can't read Japanese at all and I can't actually navigate those screens (I just press F12 which for some unknown reason to me works as "exit").

Reply:
Sure, I can create a separate task for the Japanese tests. I can't read Japanese either. I mainly just randomly fiddled with the widgets to make sure everything seemed to be working. It also looks like there's some minor functional differences between the Japanese and English tests. I should probably reconcile those.

I'll also add the method tests and examples you requested above.